packages/ncids-css/scss/components/_all.scss

```diff
@@ -38,3 +38,8 @@
 @import './usa-sidenav/nci-sidenav';
 @import './usa-site-alert/all';
 @import './usa-skipnav/usa-skipnav';
+@import 'uswds/dist/scss/components/step-indicator';
```

Comment:
Why have these been added? I had removed all components we had not worked on yet.